Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/connected_components.h

```diff
@@ -394,6 +394,10 @@ std::size_t keep_largest_connected_components(PolygonMesh& pmesh,
  *      is chosen by the user, but must be constructible from `0` and support `operator+=()` and
  *      comparisons.
  *    \cgalParamEnd
+ * *    \cgalParamBegin{dry_run}
+ *      a boolean paramter. If `true`, the operation will be performed normally. This is the default.
```

Comment:
default `false`

Reply:
Actually the sentences is mixed up I think

Reply:
Oops, yes they are